messages/pt.json

```diff
@@ -51,6 +51,7 @@
   "store/search.text": "Exibindo {from}-{to} de {recordsFiltered} resultados",
   "store/search.selected-filters": "Filtrar por:",
   "store/search.total-products": "{recordsFiltered} <span class=\"c-muted-2\">{recordsFiltered, plural, one {Produto} other {Produtos}}</span>",
+  "store/search.total-products-2": "{recordsFiltered} <span>{recordsFiltered, plural, one {Produto} other {Produtos}}</span>",
```

Comment:
why can't you just change the original?

Reply:
Since we moved from FormattedHTMLMessage to FormattedMessage if the message value had an HTML like  `foo <span>bar</span>` it would render the text `foo <span>bar</span>` and not `foo bar` as expected.